server/data/hmppsAuthClient.ts

```diff
@@ -47,6 +48,17 @@ export default class HmppsAuthClient {
     return new RestClient('HMPPS Auth Client', config.apis.hmppsAuth, token)
   }
 
+  getUserLocations(token: string): Promise<Location[]> {
+    return HmppsAuthClient.restClient(token)
+      .get({ path: '/api/users/me/locations' })
```

Comment:
I think this is the wrong endpoint... seems like it returns locations _within_ the prison, whereas we want the prisons themselves, which are found at `/api/users/me/caseLoads`, returning an array of caseload (prison) models,
```
 {
    "caseLoadId": "MDI",
    "description": "Moorland Closed (HMP & YOI)",
    "type": "INST",
    "caseloadFunction": "GENERAL",
    "currentlyActive": false
  }
```